tools/ci_build/build.py

```diff
@@ -1101,6 +1101,15 @@ def adb_shell(*args, **kwargs):
 def run_training_python_frontend_tests(cwd):
     run_subprocess([sys.executable, 'onnxruntime_test_ort_trainer.py'], cwd=cwd)
     run_subprocess([sys.executable, 'onnxruntime_test_training_unit_tests.py'], cwd=cwd)
+    run_subprocess([
```

Comment:
are the changes on this file part of this PR? It seems to be from the change for the multiple choice

Reply:
This tests were running during frontend e2e pipeline. However, they are not mixed precision and really belong to the regular CI. It is better to have them in the regular CI.

Reply:
BTW, these tests is not part of multiple choice.